styles/go.dev/catppuccin.user.css

```diff
@@ -0,0 +1,867 @@
+    @surface1: @catppuccin[@@lookup][@surface1];
+    @surface0: @catppuccin[@@lookup][@surface0];
+    @base: @catppuccin[@@lookup][@base];
+    @mantle: @catppuccin[@@lookup][@mantle];
+    @crust: @catppuccin[@@lookup][@crust];
+    @accent-color: @catppuccin[@@lookup][@@accent];
+
+    color-scheme: if(@lookup = latte, light, dark);
+
+    ::selection {
+      background-color: fade(@accent-color, 30%);
+    }
+
+    input,
+    textarea {
+      &::placeholder {
+        color: @subtext0 !important;
+      }
+    }
+
+    background-color: @base;
+
+    --abbey: @mantle;
+    --black-1: @crust;
+    --color-border: @surface2;
+    --color-background: @base;
+    --color-background-accented: @mantle;
+    --color-background-banner: @mantle;
+    --color-background-card-footer: @crust;
+    --color-background-code: @surface0; // Code Snippets
+    --color-background-inverted: @crust;
+    --color-background-logo: @text;
+    --color-background-playground-input: @mantle;
+    --color-brand-primary: @accent-color; // Affects most Go-Color Elements
+    --color-button: @accent-color;
+    --color-button-accented: @accent-color;
+    --color-button-text: @crust;
+    --color-button-text-disabled: @subtext0;
+    --color-bright-text-link: @accent-color;
+    --color-text: @text;
+    --color-text-link: @accent-color;
+    --color-text-subtle: @subtext1;
+    --focus-box-shadow: 0 0 .0625rem .0625rem @accent-color; // Only modifies color;
+    --gray-2: @surface0;
+    --gray-3: @text;
+    --gray-6: @crust;
+    --pink: @red;
+    --white: @text;
+
+    .btn {
+      background: @accent-color;
+      color: @crust;
+    }
+    // /dl page in go.dev
+    .Downloads {
+      div.filename img {
+        @svg: escape('<svg width="14" height="13" fill="none" xmlns="http://www.w3.org/2000/svg"><path d="M2.992 12.992V12q.928 0 1.408-.176.384-.127.512-.384.08-.144.08-.448H.496L0 10.496v-10L.496 0h12.992L14 .496v9.248l-1.008-1.008V.992h-12V10H6.72l-.496.496 2.496 2.496zm7.856 0 2.512-2.496-.72-.704-1.648 1.664V4.992H10v6.448L8.352 9.792l-.704.704 2.496 2.496z" fill="@{text}"/></svg>');
+        content: url('data:image/svg+xml,@{svg}')
+      }
+
+      table.downloadtable {
+        --color-background-accented: @surface0;
+      }
+    }
+    // Main Page Footer & Pkg.go Footer
+    .Footer, .go-Footer{
+      background-color: @mantle;
+
+      button[aria-label="Shorcuts Modal"] img.go-Icon {
+        @svg: escape('<svg xmlns="http://www.w3.org/2000/svg" height="24px" viewBox="0 0 24 24" width="24px" fill="@{text}"><path d="M20 5H4c-1.1 0-1.99.9-1.99 2L2 17c0 1.1.9 2 2 2h16c1.1 0 2-.9 2-2V7c0-1.1-.9-2-2-2m-9 3h2v2h-2zm0 3h2v2h-2zM8 8h2v2H8zm0 3h2v2H8zm-1 2H5v-2h2zm0-3H5V8h2zm9 7H8v-2h8zm0-4h-2v-2h2zm0-3h-2V8h2zm3 3h-2v-2h2zm0-3h-2V8h2z"/><path d="M0 0h24v24H0zm0 0h24v24H0z" fill="none"/></svg>');
+        content: url('data:image/svg+xml,@{svg}');
+        filter: none;
+      }
+
+      img.Footer-gopher {
```

Comment:
I'm seeing this "Footer-gopher" appear twice. I'm guessing we still have other duplicated SVGs. Would you mind going through and making sure none of the SVGs are duplicated?

Reply:
I've done a complete and thorough pass this time 'round. There should be no duplicates now other than ones that need a different color.